src/components/cells/cell-menu.jsx

```diff
@@ -15,8 +15,20 @@ export class CellMenuUnconnected extends React.Component {
   }
 
   render() {
+    let languagesTaskList
+    if (this.props.languages.length) {
```

Comment:
do we need the conditional here? i can't remember how this works, but I think that if you `.map` and empty array, you always get an empty array back, and that if you pass that into the jsx below it will work just as well as the undefined `languageTaskList`. does that sound right?